app/models/notification_setting.rb

```diff
@@ -28,6 +28,18 @@ class NotificationSetting < ApplicationRecord
   enum setting_type: NOTIFICATION_TYPES
   belongs_to :user
 
+  def self.get_stream_kitsu_notification_type(type, stream_mentions, feed_id)
+    stream_kitsu_notification_setting_map = {
+      follow: :follows,
+      post: :posts,
+      post_like: :likes,
+      comment_like: :likes,
+      invited: nil,
+      comment: 'mentions'.to_sym ? stream_mentions.include?(feed_id.to_i) : :posts
+    }
+    stream_kitsu_notification_setting_map[type.to_sym]
```

Comment:
This might be cleaner to express as a `case..when`

```ruby
case type
when :follow then :follows
when :post then :posts
when :post_like, :comment_like then :likes
when :comment then stream_mentions.include?(feed_id.to_i) ? :mentions : :posts
else nil
end
```